test-engineering/contract-tests/client/tests/test_merino.py

```diff
@@ -26,6 +30,81 @@ def fixture_merino_url(request) -> str:
     return request.config.option.merino_url
 
 
+@pytest.fixture(scope="session", name="kinto_step")
+def fixture_kinto_step(
```

Comment:
I like the refactored version using 3 separate fixtures. This is great! 👍🏻

Reply:
Agreed. Thank you for suggesting to break it up.